docs/design/coreclr/profiling/davbr-blog-archive/Debugging - Activation.md

```diff
@@ -15,7 +15,7 @@ Environment variables --\> Registry --\> Profiler DLL on File system.
 
 The first link in this chain is to check the environment variables inside the process that should be profiled.  If you're running the process from a command-prompt, you can just try a "set co" from the command prompt:
 
-| 
+|
 ```
 **C:\>** set co
  (blah blah, other vars beginning with "co")
```

Comment:
This is unrelated to the whitespace changes, but are the asterisk inside the code fence desired? they won't make it bold.

Also, the vertical bar in the line before the code fence, it seems extra.

Reply:
The double asterisks are result of some auto-converter. They  should be deleted.

Also, `|` around the block should be deleted.